doc/install.rst

```diff
@@ -267,16 +267,20 @@ We can use DLR to run Tensorflow 2.x saved models (including `TensorRT converted
 
 TensorFlow C library can be downloaded from `tensorflow.org <https://www.tensorflow.org/install/lang_c>`_ or built `from source <https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/lib_package/README.md>`_.
 
+TensorFlow C/C++ headers target (bazel-bin/tensorflow/include) can be built from source with ``bazel build --config=opt tensorflow:install_headers`` or can be found from Python Tensorflow installation folder - e.g. ``/usr/local/lib/python3.7/dist-packages/tensorflow/include``.
```

Comment:
TensorFlow C/C++ headers can be obtained by building the following Tensorflow target.
```
bazel build --config=opt tensorflow:install_headers

# The headers will be saved to bazel-bin/tensorflow/include
```

Another way to get TensorFlow C/C++ headers is to install Tensorflow python package using pip. The headers can be found inside the installation folder, e.g.
```
sudo pip3 install tensorflow==<tf_version>

# The headers can be found in /usr/local/lib/python3.x/dist-packages/tensorflow/include
```

Reply:
Updated the doc